### PR TITLE
Implement Client.java to retrieve client credentials from the system

### PR DESCRIPTION
**Pull request by rsampdev**

### CODE REVIEW

**Review by mahimashanware — DISMISSED** [review]
LGTM

File: project/src/main/java/com/google/autograder/servlets/auth/RequestTokenServlet.java [hunk 1]
```diff
@@ -16,13 +16,11 @@
...
 import javax.servlet.http.HttpServletResponse;
 import com.google.autograder.servlets.helpers.API;
+import com.google.autograder.servlets.helpers.Client;
 
 // @WebServlet("/requestAccessToken")
 public final class RequestTokenServlet extends HttpServlet {
 
-    private static final String CLIENT_ID = "361755208772-l0oo78304ot5ded6rb0tgbhjhrqgmc53.apps.googleusercontent.com";
-    private static final String CLIENT_SECRET = "jbzCa4-vkwu394TEk9PEnqNj";
-
```
Comment (BenLin00) [inline project/src/main/java/com/google/autograder/servlets/auth/RequestTokenServlet.java:25]: nice

**Review by BenLin00 — DISMISSED** [review]

File: project/src/main/java/com/google/autograder/servlets/helpers/Client.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    private static String getClientID() {
+        String client_id = null;
+
+        try {
+            String json = new String(Files.readAllBytes(Paths.get(RESOURCES_PATH + CREDENTIALS_PATH)));
+            JSONObject jsonObject = (JSONObject) new JSONParser().parse(json);
+            JSONObject credentials = (JSONObject) new JSONParser().parse(jsonObject.get("credentials").toString());
+            client_id = credentials.get("client_id").toString();
+        } catch(Exception e) {
+            e.printStackTrace();
```
Comment (jonathanjlin) [inline project/src/main/java/com/google/autograder/servlets/helpers/Client.java:27]: This should just throw the exception, as if there's no credentials the entire service won't be able to do anything

**Review by jonathanjlin — DISMISSED** [review]

File: project/src/main/java/com/google/autograder/servlets/helpers/Client.java (hunk 2 above)
Comment (rsampdev) [reply project/src/main/java/com/google/autograder/servlets/helpers/Client.java:27]: Ok, that makes sense, I wouldn't even be able to make API calls without it.

**Review by BenLin00 — APPROVED** [review]
lgtm

**Pull request merged**